src/runtime-rs/crates/hypervisor/src/device/topology.rs

```diff
@@ -125,6 +126,53 @@ macro_rules! unregister_pcie_device {
     };
 }
 
+/// PCIePortBusPrefix defines the naming scheme for PCIe ports.
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+pub enum PCIePortBusPrefix {
+    RootPort,
+    SwitchPort,
+    SwitchUpstreamPort,
+    SwitchDownstreamPort,
+    BridgePort,
```

Comment:
I cannot find any real user for `BridgePort` in this PR. If this is dead code, please drop it. This commit is large enough already.